_posts/guides/2021-12-21-guide_gamemode-info-parkour.md

```diff
@@ -24,9 +24,11 @@ Momentum Mod's implementation of Parkour comes from [the implementation found in
 
 ## History of Parkour
 
-Titanfall 2 was released, and boom, Parkour.
+Parkour is an athletic pursuit originating in France in the 20th century. Originally it began as a form of athletic training for soldiers called the "méthode naturelle" (natural method). These exercises were very effective at training balance, strength, and agility. The courses they would run were known as "parcours du combant", from which the word "parkour" originates. This information has nothing to do with video games, however, and is therefore worthless.
 
-TODO fill this out
+Parkour made its way into video games in 2007 when Assasin's Creed was released, and then again when Mirror's Edge was released the next year. Those early attempts laid the groundwork. Respawn then developed Titanfall 2, a game developed in the Source engine, which took lessons from all the complex first-person movement systems that had come before. It wed the ideas of other first person parkour systems with a newfound fluidity and expressiveness that allowed for more movement options at any given moment than any of the previous games.
+
```

Comment:
I'd add something explicitly stating that Titanfall 2's parkour is what Momentum is aiming to implement since we're mentioning other games in the history, just to avoid confusion